inginious/frontend/tasks.py

```diff
@@ -119,15 +126,15 @@ def get_stored_submissions(self):
     def get_evaluate(self):
         """ Indicates the default download for the task """
         return self._evaluate
-    
+
     def get_tags(self):
         """ Get the tuple of list of the task """
         return self._tags
-        
+
     def get_number_input_random(self):
         """ Return the number of random inputs """
         return self._input_random
-        
+
```

Comment:
Let's avoid these kinds of changes. The idea is to modify the least the base code, as we would have eventual problems when an integration is done with UNCode.
Rollback these "formatting" changes as they are not necessary and also it makes more difficult the review.
Do it for the files where this happened. Only format the lines that you needed to modify and the others left them as before.